.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,38 @@
+name: pull_request
+
+on:
+  push:
+    branches:
+    - 'master'
+  pull_request:
```

Comment:
does this mean on any pull request event?

Reply:
yes i think so (copy pasted from original go utils)